src/app/features/auth/registration/registration.component.ts

```diff
@@ -184,28 +184,21 @@ export class RegistrationComponent implements OnInit, OnDestroy {
 
   private handleError(error: HttpErrorResponse) {
     if (error.status === 0) {
-      // Client-side or network error
-      console.error('An error occurred:', error.error);
+      console.error('An server error occurred:', error.error);
```

Comment:
I suggest to replace it to:
```suggestion
      console.error('A server error occurred:', error.error);
```

Reply:
Done.